lamindb/_finish.py

```diff
@@ -198,10 +213,16 @@ def save_context_core(
         run.finished_at = datetime.now(timezone.utc)
 
     # track report and set is_consecutive
-    if not is_ipynb:
-        run.is_consecutive = True
+    if report_path is None:
+        run.is_consecutive = is_consecutive
         run.save()
     else:
+        if not from_cli:
+            if get_seconds_since_modified(filepath) > 2 and not ln_setup._TESTING:
+                # this can happen when auto-knitting an html with RStudio
```

Comment:
Creating a test setup for this is a bit more cumbersome, I will ship once I know this is meaningful.